python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -93,8 +97,10 @@ def start_ganga():
     logger.info("Checking Credentials")
 
     if missing_cred:
-        raise Exception("Failed due to missing credentials %s" %
-                        str(missing_cred))
+        raise Exception("Failed due to missing credentials %s" % str(missing_cred))
+
+    from Ganga.GPI import queues
+    queues.unlock()
```

Comment:
No, I see now how `queues` is constructed so this is fine for now. In future we should have a `start_queues()` and `stop_queues()` function which does something global.